### PR TITLE
Add exception_main_thread to Snuba

### PR DESCRIPTION
**Pull request by marandaneto**
### Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

### CODE REVIEW

File: snuba/snuba_migrations/events/0020_add_main_thread_column.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+
+    def forwards_ops(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.AddColumn(
+                storage_set=StorageSetKey.EVENTS,
+                table_name=table_name,
+                column=Column("exception_main_thread", bool(Modifiers(nullable=True))),
```
Comment (marandaneto) [inline snuba/snuba_migrations/events/0020_add_main_thread_column.py:18]: Does `bool(Modifiers(nullable=True))` work? didn't find any example with `bool`

File: snuba/snuba_migrations/events/0020_add_main_thread_column.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                after="modules.version",
+                target=target,
+            )
+            for table_name, target in [
+                ("errors_local", OperationTarget.LOCAL),
+                ("errors_dist", OperationTarget.DISTRIBUTED),
```
Comment (marandaneto) [inline snuba/snuba_migrations/events/0020_add_main_thread_column.py:24]: This is copy paste, assuming it's the same.

File: snuba/snuba_migrations/events/0020_add_main_thread_column.py (hunk 1 above)
Comment (marandaneto) [reply snuba/snuba_migrations/events/0020_add_main_thread_column.py:18]: Ok not really.

> error: Argument 2 to "Column" has incompatible type "bool"; expected "ColumnType[MigrationModifiers]"

What should I use here?

Comment (marandaneto) [reply snuba/snuba_migrations/events/0020_add_main_thread_column.py]: Using `UInt(8, Modifiers(nullable=True))`.

**Review by evanh — APPROVED** [review]
Looks good to me.

**Comment by marandaneto** [comment]
> Looks good to me.

Thanks @evanh 
So how does it work, should I merge it and start the other PR or this should be executed manually first by somebody?

**Comment by evanh** [comment]
Merge this PR first, then just be sure someone on SnS has manually run the migration before you merge any other PRs.

**Pull request merged**